### PR TITLE
Add Regions Bank

### PR DESCRIPTION
**Pull request by magaretha42**
I added Regions Bank.  It is a Bank in the Southern US.  They currently don't support 2 Factor Authentication.  
Please let me know if I need to change anything before it can be properly merged.

### CODE REVIEW

**Comment by RichJeanes** [comment]
Can you please delete the old logo that you placed in the `img/backup` folder?
Also, correct the new entry to have the `.png` extension, not `.img`.

**Comment by magaretha42** [comment]
Ok.  I made the changes and the build completed successfully.  Thanks for letting me know what was wrong.

File: _data/banking.yml [hunk 1]
```diff
@@ -375,6 +375,12 @@ websites:
       url: http://www.rbcroyalbank.com
       twitter: RBC_Canada
       img: rbc.png
+      tfa: No    
+    
+    - name: Regions Bank
+      url: http://www.regions.com
```
Comment (Carlgo11) [inline _data/banking.yml]: Please use `https://www.regions.com/` instead.

**Comment by RichJeanes** [comment]
Thank you for contributing! Your changes have been merged into the main branch but will not appear on the live website until the `master` branch is merged into the `gh-pages` branch, which happens periodically. There is nothing left here for you to do.

**Pull request merged**